src/controllers/Server.ts

```diff
@@ -27,6 +27,17 @@ export class ServerController extends BaseController {
       .then(response => response.result.exists);
   }
 
+  /**
+   * Returns the Kuzzle capabilities
```

Comment:
Can be add a more complete doc about what is capabilities ?

It's maybe really obvious, but it could be nice to add an example of result in the doc.

Reply:
something like that ?
```js
  /**
   * Returns the Kuzzle capabilities
   * @param {Object} options - {queuable: Boolean(true)}
   * @example {
   *   "status": 200,
   *   "error": null,
   *   "action": "getConfig",
   *   "controller": "server",
   *   "result": {
   *     "limits": {
   *       "concurrentRequests": 100,
   *       "documentsFetchCount": 10000,
   *       ...
   *       },
   *       "plugins": {
   *         "common": {
   *           "bootstrapLockTimeout": 5000,
   *           ...
   *         }
   *       },
   *       ...
   *       "version": "2.16.6"
   *   },
   *   "status": 200,
   *   "volatile": null
   * }
   * @returns {Promise<Object>}
   */
```
But I'm afraid it's too much, because the user can find the action doc [here](https://docs.kuzzle.io/core/2/api/controllers/server/capabilities/)

Reply:
I don't know what are the guidelines for the doc just now, but if it's too long, maybe just a link to https://docs.kuzzle.io/core/2/api/controllers/server/capabilities/#response can be enough ?

Reply:
So I wrote the link in the `@example`